signer/src/ecdsa.rs

```diff
@@ -161,9 +165,9 @@ impl Keypair {
         // From sp_core::ecdsa::sign_prehashed:
         let wrapped = Message::from_digest_slice(&message_hash).expect("Message is 32 bytes; qed");
         let recsig: RecoverableSignature =
-            SECP256K1.sign_ecdsa_recoverable(&wrapped, &self.0.secret_key());
+            Secp256k1::signing_only().sign_ecdsa_recoverable(&wrapped, &self.0.secret_key());
```

Comment:
yeah ok, this probably because they some `lazy_static` stuff that doesn't compile for no-std?

then removing global-context makes sense